src/NUnitFramework/framework/Interfaces/TNode.cs

```diff
@@ -400,7 +448,7 @@ public bool Pass(TNode node)
             }
         }
 
-        #endregion
+#endregion
```

Comment:
```suggestion
        #endregion
```
To match the `#region` start (I know that the VS tooling for this is lacking :()